src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -2666,6 +2667,13 @@ private class VirtualizingPanelSmoothScroller : LinearSmoothScroller
 		{
 			private const float BaseDuration = 250f, ScalableDuration = 150f; // in ms
 
+			private const int TARGET_SEEK_SCROLL_DISTANCE_PX = 10000;
+
+			// Trigger a scroll to a further distance than TARGET_SEEK_SCROLL_DISTANCE_PX so that if target
+			// view is not laid out until interim target position is reached, we can detect the case before
+			// scrolling slows down and reschedule another interim target scroll
```

Comment:
```suggestion
			// Trigger a scroll to a further distance than TARGET_SEEK_SCROLL_DISTANCE_PX so that if the target
			// view is not laid out until the interim target position is reached, we can detect the case before
			// scrolling slows down and reschedules another interim target scroll
```